mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,359 @@
+"""Conversion tool from Neuroscan CNT to FIF
+"""
+
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+from os import path
+import datetime
+import calendar
+
+import numpy as np
+
+from ...utils import warn, verbose
+from ...channels.layout import _topo_to_sphere
+from ..constants import FIFF
+from ..utils import _mult_cal_one, _find_channels, _create_chs
+from ..meas_info import _empty_info
+from ..base import _BaseRaw, _check_update_montage
+from ..utils import read_str
+
+
+def read_raw_cnt(input_fname, montage, eog=(), misc=(), ecg=(), emg=(),
+                 preload=False, verbose=None):
+    """Read CNT data as raw object.
+
+    .. Note::
+    If montage is not provided, the x and y coordinates are read from the file
```

Comment:
did you build doc? This almost certainly needs to be indented to render properly